src/commands/ghCommands/status.js

```diff
@@ -0,0 +1,68 @@
+const GitHubApi = require('github');
+const chalk = require('chalk');
+const github = GitHubApi();
+
+module.exports = {
+	command: 'status',
+	description: 'Set a PR status',
+	builder: yarg =>
+		yarg
+			.options({
+				value: {
+					alias: 's', // s for 'status'. -v is conventionally a 'version' alias
+					demandOption: true,
+					type: 'string',
+					describe: 'The status value',
+					choices: ['pending', 'failure', 'error', 'success'],
+				},
+				commit: {
+					alias: 'c',
+					demandOption: true,
+					type: 'string',
+				},
+				token: {
+					alias: 't',
+					type: 'string',
+				},
+				repo: {
+					alias: 'r',
+					demandOption: true,
+					type: 'string',
+				},
+				context: {
+					demandOption: true,
+					type: 'string',
+				},
+				description: {
+					alias: 'd',
+					type: 'string',
+				},
+			})
+			.check((argv, opts) => {
+				if (argv.commit.length < 40) {
```

Comment:
Does the GH api demand the full SHA-1?

(I am assuming yes, but slightly surprised)

Reply:
yeah, unfortunately